persist/sqldb/migrate.go

```diff
@@ -48,9 +48,9 @@ func migrate(cfg migrateCfg, session sqlbuilder.Database) error {
     workflow text,
     startedat timestamp,
     finishedat timestamp,
-    primary key (id, namespace)
+    primary key (id, namespace),
+    constraint idx_name unique(name)
 )`,
-		`create unique index if not exists idx_name on ` + cfg.tableName + ` (name)`,
```

Comment:
rather than modify the statement above - which will require manual migration for current users - can you just remove the `if not exists` and run that?

Reply:
Do you mean we already have public releases including database schema?

Reply:
@alexec - done.